src/NHSD.BuyingCatalogue.Ordering.Persistence/Repositories/OrderRepository.cs

```diff
@@ -56,31 +55,31 @@ public async Task<string> CreateOrderAsync(Order order)
                 throw new ArgumentNullException(nameof(order));
             }
 
-            using (var dbContextTransaction = _context.Database.BeginTransaction())
+            using (var dbContextTransaction = _context.Database.BeginTransactionAsync())

```

Comment:
I think we probably want an `await` here.

Reply:
👍